orthoinference/src/main/java/org/reactome/orthoinference/SpeciesCheck.java

```diff
@@ -0,0 +1,65 @@
+package org.reactome.orthoinference;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.schema.InvalidAttributeException;
+
+public class SpeciesCheck {
+
+	// Determines if there is a species attribute in any constituent instances of entityInst.
+	// Unless its an 'OtherEntity' (which will return false), the function will check the instance or iterate on it's
+	// sub-instances until it finds an existing 'species' attribute, or else it will return false.
+	public static boolean hasSpecies(GKInstance entityInst) throws InvalidAttributeException, Exception
+	{
+		if (entityInst.getSchemClass().isa(ReactomeJavaConstants.OtherEntity))
+		{
+			return false;
+		} else if (entityInst.getSchemClass().isa(ReactomeJavaConstants.EntitySet)) // || entityInst.getSchemClass().isa(ReactomeJavaConstants.Polymer) || entityInst.getSchemClass().isa(ReactomeJavaConstants.EntitySet))
+		{
+			for (Object member : entityInst.getAttributeValuesList(ReactomeJavaConstants.hasMember))
```

Comment:
`member` is probably always going to be a `GKInstance`, so you shouldn't need to use `Object` here.